doc/devdocs/modules/mousewithoutborders.md

```diff
@@ -0,0 +1,22 @@
+# Mouse Without Borders module
+This file contains the documentation for the Mouse Without Borders PowerToy module.
+## Table of Contents:
+- [Mouse Without Borders module](#mouse-without-borders-module)
+  - [Table of Contents](#table-of-contents)
+  - [Status colors](#status-colors)
+
+## Status colors
+The following colors are used to indicate the connection status to the user when trying to connect to another computer:
+
+| Connection Status | Color    | Hex Code    |
+| :-----: | :---: | :---: |
+| NA | Dark Grey   | `#00717171`  |
```

Comment:
Same as on the docs PR: What means NA?

Reply:
It means SocketStatus.NA, which I assume means the information about the socket status is not available